dot/parachain/dispute/overseer/message.go

```diff
@@ -0,0 +1,83 @@
+package overseer
+
+import (
+	parachainTypes "github.com/ChainSafe/gossamer/dot/parachain/types"
+	"github.com/ChainSafe/gossamer/lib/common"
+)
+
+type ChainAPIMessage struct {
+	RelayParent     common.Hash
+	ResponseChannel chan *uint32
+}
+
+type PersistedValidationData struct {
+	ParentHead             []byte
+	RelayParentNumber      uint32
+	RelayParentStorageRoot common.Hash
+	MaxPOVSize             uint32
+}
+
+type AvailableData struct {
+	POV            []byte
+	ValidationData PersistedValidationData
+}
```

Comment:
We already have [AvailableData](https://github.com/ChainSafe/gossamer/blob/f9b3281b3e1798510948499b6a42900334a97edc/dot/parachain/available_data_fetching.go#L52)

Reply:
The types in the overseer is temporary. Please see the comment on top of the file.